python/mxnet/gluon/loss.py

```diff
@@ -706,3 +706,77 @@ def hybrid_forward(self, F, pred, positive, negative):
                      axis=self._batch_axis, exclude=True)
         loss = F.relu(loss + self._margin)
         return _apply_weighting(F, loss, self._weight, None)
+
+class CosineEmbeddingLoss(Loss):
+    r"""For a target label 1 or -1, vectors target and pred, the function computes the cosine distance
+    between the vectors. This can be interpretted as how similar/dissimilar two input vectors are.
+
+    .. math::
+        L = \begin{gather*}
```

Comment:
isn't the loss a cumulative sum over all data points? so the formula should begin with `\sum_i`